src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -275,55 +279,79 @@ public void shutdown() {
-        // opportunistically check whether the combined pileup has no evidence of variation, which we will define as
-        // having at most one variant read
-        final int totalNonRef = countNonRef(refBase, context);
-        if (totalNonRef < MTAC.minVariantsInPileup) {
+        if( context == null || context.getBasePileup().isEmpty() ) {
             return new ActivityProfileState(refInterval, 0.0);
         }
 
-        final Map<String, AlignmentContext> splitContexts = context.splitContextBySampleName(header);
-        final AlignmentContext tumorContext = splitContexts.get(MTAC.tumorSampleName);
-        final AlignmentContext normalContext = splitContexts.get(MTAC.normalSampleName);
-
-        // if there are no tumor reads... there is no activity!
-        if (tumorContext == null) {
-            return new ActivityProfileState(refInterval, 0);
-        }
-
-        final int tumorNonRef = countNonRef(refBase, tumorContext);
-        if (tumorNonRef < MTAC.minVariantsInPileup) {
-            return new ActivityProfileState(refInterval, 0.0);
-        }
+        final ReadPileup pileup = context.getBasePileup();
+        final ReadPileup tumorPileup = pileup.getPileupForSample(MTAC.tumorSampleName, header);
+        final Pair<Integer, Double> tumorAltCountAndQualSum = altCountAndQualSum(tumorPileup, refBase);
+        final int tumorAltCount = tumorAltCountAndQualSum.getFirst();
+        final int tumorRefCount = tumorPileup.size() - tumorAltCount;
 
-        // since errors are rare, the number of errors (if reads are independent) is approximately a Poisson random variable,
-        // with mean equal to its variance
-        final double expectedTumorNonRefDueToError = StreamSupport.stream(tumorContext.getBasePileup().spliterator(), false)
-                .mapToDouble(pe -> QualityUtils.qualToErrorProb(pe.getQual()))
-                .sum();
-        final double tumorNonRefStdev = Math.sqrt(expectedTumorNonRefDueToError);
+        final double tumorLog10Odds = -QualityUtils.qualToErrorProbLog10(tumorAltCountAndQualSum.getSecond()) +
+                MathUtils.log10Factorial(tumorAltCount) + MathUtils.log10Factorial(tumorRefCount) - MathUtils.log10Factorial(tumorPileup.size() + 1);
 
-        if (tumorNonRef < expectedTumorNonRefDueToError + MTAC.tumorStandardDeviationsThreshold * tumorNonRefStdev) {
+        if (tumorLog10Odds < MTAC.initialTumorLodThreshold) {
             return new ActivityProfileState(refInterval, 0.0);
+        } else if (hasNormal()) {
+            final ReadPileup normalPileup = pileup.getPileupForSample(MTAC.normalSampleName, header);
+            final Pair<Integer, Double> normalAltCountAndQualSum = altCountAndQualSum(normalPileup, refBase);
+            final int normalAltCount = normalAltCountAndQualSum.getFirst();
+            final double normalQualSum = normalAltCountAndQualSum.getSecond();
+            if (normalAltCount > normalPileup.size() * 0.3 && normalQualSum > 100) {
```

Comment:
In this case the qual sum turns out to be a term in a fairly principled approximation to a likelihood-based calculation.  I'm still against it standing on its own as a heuristic.  I'll run the calculation by you in front of a whiteboard.